src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -165,37 +189,46 @@ void testGetAppealById_returnsEmptyAppeal() {
 
     @Test
     void testGetAppealsByPenaltyReference_returnsListOfAppeals() {
-
-        when(appealRepository.findByPenaltyReference(any(String.class))).thenReturn(List.of(createAppealEntity(TestData.Appeal.PenaltyIdentifier.penaltyReference)));
-        when(appealMapper.map(any(AppealEntity.class))).thenReturn(createAppeal());
-
-        List<Appeal> appealList = appealService.getAppealsByPenaltyReference(TestData.Appeal.PenaltyIdentifier.penaltyReference);
+        ReasonEntity reasonEntity = createReasonEntityWithOther();
+        Reason reason = createReasonWithOther();
+        when(appealRepository.findByPenaltyReference(any(String.class))).thenReturn(
+            List.of(createAppealEntity(TestData.Appeal.PenaltyIdentifier.penaltyReference, reasonEntity)));
+        when(appealMapper.map(any(AppealEntity.class))).thenReturn(createAppeal(reason));
+
+        List<Appeal> appealList = appealService.getAppealsByPenaltyReference(
+            TestData.Appeal.PenaltyIdentifier.penaltyReference);
         Appeal appeal = appealList.get(0);
 
         assertEquals(1, appealList.size());
 
-        assertEquals(TestData.Appeal.PenaltyIdentifier.penaltyReference, appeal.getPenaltyIdentifier().getPenaltyReference());
+        assertEquals(TestData.Appeal.PenaltyIdentifier.penaltyReference,
+            appeal.getPenaltyIdentifier().getPenaltyReference());
         assertEquals(TestData.Appeal.PenaltyIdentifier.companyNumber, appeal.getPenaltyIdentifier().getCompanyNumber());
         assertEquals(TestData.Appeal.Reason.OtherReason.title, appeal.getReason().getOther().getTitle());
         assertEquals(TestData.Appeal.Reason.OtherReason.description, appeal.getReason().getOther().getDescription());
         assertEquals(2, appeal.getReason().getOther().getAttachments().size());
-        assertEquals(TestData.Appeal.Reason.Attachment.id, appeal.getReason().getOther().getAttachments().get(0).getId());
-        assertEquals(TestData.Appeal.Reason.Attachment.name, appeal.getReason().getOther().getAttachments().get(0).getName());
-        assertEquals(TestData.Appeal.Reason.Attachment.contentType, appeal.getReason().getOther().getAttachments().get(0).getContentType());
-        assertEquals(TestData.Appeal.Reason.Attachment.size, appeal.getReason().getOther().getAttachments().get(0).getSize());
+        assertEquals(TestData.Appeal.Reason.Attachment.id,
+            appeal.getReason().getOther().getAttachments().get(0).getId());
+        assertEquals(TestData.Appeal.Reason.Attachment.name,
+            appeal.getReason().getOther().getAttachments().get(0).getName());
+        assertEquals(TestData.Appeal.Reason.Attachment.contentType,
+            appeal.getReason().getOther().getAttachments().get(0).getContentType());
+        assertEquals(TestData.Appeal.Reason.Attachment.size,
+            appeal.getReason().getOther().getAttachments().get(0).getSize());
```

Comment:
This one can be simplified to:

```
@DisplayName("Get multiple appeals by penalty reference")
@Test
void testGetAppealsByPenaltyReference() {
    String penaltyReference = "ABC";
    Appeal appeal = createAppeal(createReasonWithOther());
    AppealEntity appealEntity = createAppealEntity(createReasonEntityWithOther());

    when(appealRepository.findByPenaltyReference(penaltyReference).thenReturn(List.of(appeal));
    when(appealMapper.map(appealEntity).thenReturn(appeal);

    List<Appeals> appeals = appealService.getAppealsByPenaltyReference(penaltyReference);
    assertEquals(1, appeals.size());
    assertEquals(appeal, appeals.get(0));
}
```

Reply:
Test constants were used to simplify the unit tests. Refactoring to remove unnecessary assertions has been added into a unit test refactoring task. This will be resolved separately to this PR.